app/frontend/src/search/index.js

```diff
@@ -128,3 +128,7 @@ renderAutocomplete({
     updateUrlQueryParams('location', value, window.location.href);
   },
 });
+
+document.getElementById('location').addEventListener('input', (e) => {
```

Comment:
What happens if this element isn't there? It probably should be, but this could _potentially_ be executed before the DOM has loaded couldn't it?

Reply:
doesnt do any harm, whilst im not sure totally neccesary as it use betElementById, im sure ie9 or something might say 'why didnt you just do it'?

Reply:
actually this does work